prodsys/simulation/state.py

```diff
@@ -391,6 +407,7 @@ def process_state(self, target: List[float], initial_transport_step: bool, last_
             except exceptions.Interrupt:
                 if not self.interrupted:
                     raise RuntimeError(f"Simpy interrupt occured at {self.state_data.ID} although process is not interrupted")
+        yield from self.handle_loading(self.handling_time, "loading")
```

Comment:
this needs to happen with a try except Interrupt inside an infite while loops that is broken when the loading is over (same loading time again as at start when interrupt happens)